anthos-onprem-terraform/avmw_user_cluster_metallb/README.md

```diff
@@ -50,3 +50,25 @@ have changed directory to where this samples is:
 
     You can view your user cluster in the
     [Anthos clusters page](https://console.cloud.google.com/anthos/clusters).
+
+### Upgrade the user cluster with terraform
+
+Before upgrading the user cluster, please make sure the admin cluster of the target version has been upgraded. The steps to upgrade the admin cluster is listed in the [public documentation](https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/upgrading#:~:text=In%20the%20Cluster%20basics%20section,Click%20Upgrade.).
```

Comment:
I'd change the sentence to the following:
Before upgrading the user cluster, please make sure the admin cluster platform controller has been upgraded to the target version. The steps to upgrade the admin cluster is listed in the [public documentation]. 

Also for the public documentation doc use https://cloud.google.com/anthos/clusters/docs/on-prem/latest/how-to/upgrade-on-prem-api#available_versions_for_upgrades

Reply:
Thanks for the suggestion. Updated accordingly.